manifests/server/dbgroup.pp

```diff
@@ -46,7 +46,7 @@
   }
 
   postgresql_psql { "${title}: CREATE GROUP ${groupname}":
-    command     => "CREATE GROUP '${groupname}'",
+    command     => "CREATE GROUP ${groupname}",
```

Comment:
Since `${groupname}` can't be quoted, should we add a defensive check here that checks if `${groupname}` has any whitespace?